src/parsers/RBDyn/parsers/common.h

```diff
@@ -23,7 +23,7 @@ namespace rbd
 namespace parsers
 {
 
-enum class RBDYN_PARSERS_DLLAPI ParserInput
+enum class ParserInput
```

Comment:
I had compilation issues in CI because of this. It seems that not all compilers support this well and from what I gathered, it is not necessary (see https://stackoverflow.com/a/40402637/6458469 and https://stackoverflow.com/a/18641809/6458469)

Reply:
Oh ok, it seems Windows is happy to build without it so that's probably fine :)